### ISSUE TITLE
🐛 [BUG/ERROR]: Targeting S+ (version 31 and above) requires that an explicit value for android:exported be defined when intent filters are present]

### ISSUE THREAD

**Issue by mahdi-malv** [tapsellorg/TapsellPlusSDK-FlutterPlugin#12]
**Describe the issue**:
Runing the app leads to this error:

```
Targeting S+ (version 31 and above) requires that 
  an explicit value for android:exported be defined when intent filters are present]
```

**How to reproduce this issue:**
- Run the sample

**App package name**: Not important

**Comment by mahdi-malv** [comment]
**Suggestion**: To understand where the issue is caused, run the app (using flutter) and view **android logcat**
Then filter with `Error` and stop when `Targeting S+ ...` was printed. The stacktrace guides you to it.

`android:exported` is commonly caused by old template of flutter which does not include this for **Main Activity** manifest. Fix it by adding it manually:

```xml
<activity
     android:name=".MainActivity"
     android:exported="true"
     android:launchMode="singleTop"
     android:theme="@style/LaunchTheme"
 </activity>
```

Ref: https://stackoverflow.com/questions/70333565/targeting-s-version-31-and-above-requires-that-an-explicit-value-for-android

**Comment by mahdi-malv** [comment]
Issue fully fixed in 2.1.7
